3600/3/app/views/layout.erb

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
```

Comment:
You have layout file but you don't use it. Don't do so

Reply:
Removed

Reply:
Without layout file:
![Screenshot from 2019-08-02 15-54-03](https://user-images.githubusercontent.com/46228563/62371701-2b443180-b53e-11e9-9c24-25e888ccdf86.png)
![Screenshot from 2019-08-02 15-55-17](https://user-images.githubusercontent.com/46228563/62371737-4151f200-b53e-11e9-981d-fff08c00cedb.png)

Reply:
To use layout
```ruby
erb :index, layout: :layout
```